exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,12 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if(mark>80)return "A";
+  else if(mark<80 && mark>60)return "B";
+  else if(mark<=60 && mark>=50)return "C";
+  else return "F";
```

Comment:
Great job on providing a working solution.

However, please code formating is a good practice as it helps code debugging in case something goes wrong.